From 566b23e50542f8b727258b4abd2037b512c83c56 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Mon, 14 Jan 2019 20:58:10 +0900
Subject: [PATCH 1/4] shared: define interval time to retry connection

This update is to define a common const `CONN_RETRY_USEC` for defining
interval time to retry connection in micro sec. The default value is
1,000,000 usec (= 1 sec).

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/shared/common.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/shared/common.h b/src/shared/common.h
index cde013e..e244db9 100644
--- a/src/shared/common.h
+++ b/src/shared/common.h
@@ -23,6 +23,9 @@
 
 #define NO_FLAGS 0
 
+/* Interval time to retry connection. */
+#define CONN_RETRY_USEC (1000 * 1000)  /* micro sec */
+
 /*
  * When doing reads from the NIC or the client queues,
  * use this batch size
-- 
2.17.1

